CHANGELOG.md

```diff
@@ -1,7 +1,47 @@
+# v0.6.0
+
+**Note**: Changed username identity format to username!Domain, username now specified in the OtherName SAN. If you have deployed your own instance of Fulcio and are using username issuers, you must update to the latest Cosign release.
```

Comment:
Lgtm, but we also need to cut a new cosign too

Reply:
cosign is ready to be cut? cc @dlorenc

Reply:
Sure!